### PR TITLE
Moved button scripts to CommonScripts and added button tags

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
In Progress

## Related Issues
Fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5188
Related to: https://jira-hq.paloaltonetworks.local/browse/CIAC-5042

## Description
Moved the following scripts to the CommonScripts pack:
- LinkIncidentsButton
- AssignToMeButton
- GenerateSummaryReportButton
- AppendindicatorFieldWrapper
- IsolationAssetWrapper

Additionally, added the `incident-action-button` tag to the scripts that were missing it, and also to `Base64EncodeV2` and to `Base64Decode`.

## Does it break backward compatibility?
Yes

### CODE REVIEW

File: Packs/CaseManagement-Generic/ReleaseNotes/1_4_2.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+##### CaseManagement-Generic
```
Comment (Shellyber) [inline Packs/CaseManagement-Generic/ReleaseNotes/1_4_2.md:1]: Need to fix the release notes according to our template
Change the headers and the contents as follows:
```
#### Scripts
##### Script name
- Note: Moved to the *pack-name* pack.
```

**Review by Shellyber — COMMENTED** [review]
Nice Ido. 
Please see my comment
Also, you'll need to generate and bump the version to CommonScripts as well

Does the pack CaseManagement should be deprecated? is something left in it?

**Comment by idovandijk** [comment]
> 

Thanks, currently looking into an issue that prevented me from updating CommonScripts.
I will update both RN.
CaseManagement still has scripts in it so it should not be deprecated.

File: Packs/CaseManagement-Generic/ReleaseNotes/1_4_2.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+#### Scripts
+##### AssignToMeButton
+- Note: Moved to the *CommonScripts* pack.
+##### GenerateSummaryReportButton
+- Note: Moved to the *CommonScripts* pack.
+- ##### LinkIncidentsButton
```
Comment (Shellyber) [inline Packs/CaseManagement-Generic/ReleaseNotes/1_4_2.md:7]:
```suggestion
##### LinkIncidentsButton
```

File: Packs/CommonScripts/ReleaseNotes/1_11_6.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+
+#### Scripts
+##### AssignToMeButton
+- Note: Moved from the *CaseManagement-Generic* pack to this pack.
+##### GenerateSummaryReportButton
+- Note: Moved from the *CaseManagement-Generic* pack to this pack.
+- ##### LinkIncidentsButton
```
Comment (Shellyber) [inline Packs/CommonScripts/ReleaseNotes/1_11_6.md:7]:
```suggestion
##### LinkIncidentsButton
```

**Review by Shellyber — COMMENTED** [review]
nice!

**Comment by idovandijk** [comment]
@DeanArbel since Shelly doesn't have the capacity at the moment to review this, can you please help with the review?

**Review by RotemAmit — COMMENTED** [review]
Nice work! 
I  have some general comments:
* Rerun update release notes command for the packs CommonScripts and Filters and transformers in order to add the docker image update to the release notes.
* Add to the RN of CommonScripts all the things you moved to it, like AppendindicatorFieldWrapper and IsolationAssetWrapper.
* Update the docker image where it is needed
* Add to the .pack-ignore file in the original pack the errors, RN114 in a format like this: [file:1_0_2.md]
ignore=RN113,RN114
we can ignore them because this is what we want to do, to move the scripts to a different place.

**Comment by idovandijk** [comment]
PR is ready, blocked by https://jira-hq.paloaltonetworks.local/browse/CIAC-5400

**Review by Shellyber — APPROVED** [review]
Nice

**Comment by idovandijk** [comment]
@anara123 Can you please force merge this? BA114 cannot be fixed since I moved scripts between packs

**Comment by anara123** [comment]
@idovandijk please resolve the conflict first

**Comment by idovandijk** [comment]
> @idovandijk please resolve the conflict first

Damn, the conflict was not there before 🤦‍♂️ fixing it now.

**Pull request merged**